app/scripts/modules/cloudfoundry/src/serverGroup/configure/wizard/sections/basicSettings.cf.tsx

```diff
@@ -142,28 +124,15 @@ class BasicSettingsImpl extends React.Component<
             Start on creation <HelpField id="cf.serverGroup.startApplication" />
           </div>
           <div className="checkbox checkbox-inline">
-            <input
-              type="checkbox"
-              checked={this.props.formik.values.startApplication}
-              onChange={this.startApplicationUpdated}
-            />
+            <input type="checkbox" checked={values.startApplication} onChange={this.startApplicationUpdated} />
```

Comment:
Can we take advantage of what @jmelchio now knows about using the formik `Field` element?

Reply:
👍